src/app/models/user/user.go

```diff
@@ -1,8 +1,8 @@
 package user
 
-import (
-	"github.com/EaseApp/web-backend/src/config/db"
+/*
```

Comment:
For usage, I plan on having the models have a db client as a dependency, then use that for db access.